index.html

```diff
@@ -1,30 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <title>Save The Bay</title>
-
-    <!-- Latest compiled and minified CSS -->
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
-
-    <link rel="stylesheet" href="styles/main.css" />
-  </head>
-  <body>
-    <div class="container">
-      <div class="jumbotron">
-        <h1>Save the Bay</h1>
-      </div>
+<head>
+  <title>Save the Bay</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
```

Comment:
What about pasting copy for website, wouldn't those be hard to format? I just set sublime to auto-wrap.